MacrobenchmarkSample/app/src/main/java/androidx/benchmark/macro/target/TrivialStartupActivity.kt

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
Suggest removing this activity, and only having 2 macrobenchmarks, startup and scroll for the one activity

(also remove related resources and macrobenchmark)